crates/claims/crates/data-integrity/core/src/options.rs

```diff
@@ -13,8 +13,8 @@ pub struct ProofOptions<M, T> {
     pub context: Option<ssi_json_ld::syntax::Context>,
 
     /// Date a creation of the proof.
-    #[serde(default = "xsd_types::DateTime::now")]
-    pub created: xsd_types::DateTime,
+    #[serde(default = "created_default", skip_serializing_if = "Option::is_none")]
+    pub created: Option<xsd_types::DateTime>,
```

Comment:
I see the spec also specifies that the type of `created` should be an XSD `dateTimeStamp` and not `dateTime` (which has an optional timezone so it can be ambiguous). But maybe that's for another PR.

Reply:
Nevermind, VCDM 1.1 has `issuanceDate` as `dateTime` and we need to convert it to `dateTimeStamp` but I don't see a straightforward way to do that (I don't think we can assume UTC when there's no offset).